octopuslite/reader.py

```diff
@@ -175,7 +184,7 @@ def _parse_files(self):
             else:
                 max, min = 200, 2
```

Comment:
I don't like this hardcoding of these values.

Reply:
tbh, im not sure if i need this function anymore as i changed the pipeline to remove blank frames from the image folder in the alignment step. i can keep as an if a tuple is provided but if u really not a fan of this feature i could probs remove it

Reply:
I would remove it if that's the case

Reply:
actually i would like to keep to view filtered images before i run any analysis if that is okay